drivers/bridge/bridge.go

```diff
@@ -1043,6 +1049,54 @@ func (d *driver) CreateEndpoint(nid, eid string, ifInfo driverapi.InterfaceInfo,
 		}
 	}
 
+	// The bridge proxy arp feature requires three things to happen:
+	// 1) Set the proxyarpwifi flag on the container side bridge port to disable ARP packet flooding.
+	// 2) Generate an arp entry in the host's ARP table mapping the containers IP address to its MAC.
+	// 3) Generate a static FDB entry for the containers MAC address.
+
+	if config.EnableBrProxyArp {
+
+		BridgeIF, err := d.nlh.LinkByName(config.BridgeName)
+		if err != nil {
+			return fmt.Errorf("could not find interface with destination name %s: %v", config.BridgeName, err)
+		}
+
+		HostIF, err := d.nlh.LinkByName(hostIfName)
```

Comment:
Please remove this, `host` variable already contains the host side veth end link. In fact later you use that
```
err = d.nlh.LinkSetBrProxyArpWiFi(host, true)
```